test/mpi/test_alltoall.cc

```diff
@@ -29,24 +31,38 @@ TestAlltoall::TestAlltoall(size_t _msgsize, ucc_test_mpi_inplace_t _inplace,
         return;
     }
 
-    UCC_CHECK(ucc_mc_alloc(&rbuf_mc_header, _msgsize * nprocs, _mt));
-    rbuf       = rbuf_mc_header->addr;
+    if (!is_onesided) {
+        UCC_CHECK(ucc_mc_alloc(&rbuf_mc_header, _msgsize * nprocs, _mt));
+        rbuf = rbuf_mc_header->addr;
+    } else {
+        sbuf     = buffers[MEM_SEND_SEGMENT];
```

Comment:
the memory type paramter is ignored here, maybe return not supported if mt != UCC_MEMORY_TYPE_HOST

Reply:
i believe this is done